src/azul/transformer.py

```diff
@@ -76,12 +78,23 @@ class DocumentCoordinates(NamedTuple):
 FieldTypes = Mapping[str, FieldTypes1]
 
 
+class VersionType(Enum):
+    # No versioning; document is created or overwritten as needed
+    none = 1
```

Comment:
Since the values of these constants are arbitrary, I would avoid writing them out. See https://docs.python.org/3.6/library/enum.html#omitting-values

Reply:
While I agree, @hannes-ucsc requested writing them explicitly.

Reply:
Maybe we should do what @noah-aviel-dove suggests. I thought using auto() for a three-item enum is overkill but this sets a bad precedent. We'd have to define in CONTRIBUTING when to use auto() and when not to. Explicitly assigning a value suggests that the value is significant, which I don't think it is in this case.